src/components/UserManagement/UserTableHeader.jsx

```diff
@@ -47,7 +47,12 @@ const UserTableHeader = React.memo(props => {
       <th scope="col" id="usermanagement_resume_date">
         End Date
       </th>
-      <th scope="col" id="usermanagement_delete"></th>
+      {((
+        props.authRole !=="Owner" && 
+        props.roleSearchText !=="Owner") || 
+        props.authRole =="Owner") && (
```

Comment:
Seems this is the same condition as in `UserTableSearchHeader` lines from 56 to 58.
How about moving this into a helper function that'll be imported into both files and receives the thee values as params?
`(authRole, roleSearchText, authRole) => ((authRole !== "Owner" && roleSearchText !== "Owner") || authRole === "Owner");`

Reply:
Moved the same condition from`UserTableHeader` and `UserTableSearchHeader` into a helper function and imported in both files.